src/app/shared/services/skin/skin.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
+import { LocalStorageService } from "src/app/shared/services/local-storage/local-storage.service";
+import { APP_CONSTANTS } from "src/app/data";
+
+const { APP_FIELDS } = APP_CONSTANTS;
+
+// TODO - Where should these live and be added to? Should ultimately come from sheets?
+// Could live at deployment level, e.g. app_constants.compatible_skins
+const SKINS = ["default", "modular"];
```

Comment:
I think we need to clarify what level skins live at. Do themes dictate the skins available? Or deployments?